src/function/handler_context.ts

```diff
@@ -11,6 +11,11 @@ export interface HandlerContext {
   awsRequestId: string
   logGroupName: string
   logStreamName: string
+  /**
+   * A Promise that allows the function to respond immediately while still waiting
+   * for the operation to complete asynchronously.
+   */
+  waitUntil: Promise<any>
```

Comment:
I think this is currently an incorrect type definition, as waitUntil is a method and not itself a Promise

I've made a suggested change below:

```suggestion
  /**
   * The waitUntil method enables the application to carry on executing after the response has been sent.
   * The method has to be called at-least once synchronously before the response has beent sent, but after that it can be called multiple times, and will hold the keep the application running until all the promises passed to it have settled.
   */
  waitUntil(f: Promise<any>): void;
```

Reply:
Currently I have to pass a promise, rather than a function, to get this to work otherwise I get `"i.catch is not a function"`